my_list.rb

```diff
@@ -0,0 +1,16 @@
+require './MyEnumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list.to_a
+  end
+end
```

Comment:
- [ ] According to requirements, your `MyList` class should implement `#each` method that yields all members of the `@list` instance variable in succession 👍